django/db/backends/oracle/introspection.py

```diff
@@ -78,7 +78,17 @@ def get_table_description(self, cursor, table_name):
             name = force_text(desc[0])  # cx_Oracle always returns a 'str'
             internal_size, default = field_map[name]
             name = name % {}  # cx_Oracle, for some reason, doubles percent signs.
-            description.append(FieldInfo(*(name.lower(),) + desc[1:3] + (internal_size,) + desc[4:] + (default,)))
+            description.append(FieldInfo(*(
+                (name.lower(),) +
+                desc[1:3] +
+                (
```

Comment:
I think this is just as readable on a single line: `(internal_size, desc[4] or 0, desc[5] or 0,) +`